<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_enable">
       <property name="text">
        <string>可执行文件中写入版本信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>版本信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_company_name">
              <property name="text">
               <string>公司名称：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_product_introduct">
              <property name="text">
               <string>产品名称：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_file_introduct">
              <property name="text">
               <string>文件描述：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_law_copyright">
              <property name="text">
               <string>法律版权：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_law_trademark">
              <property name="text">
               <string>法律商标：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_orig_filename">
              <property name="text">
               <string>原始文件名：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_comment">
              <property name="text">
               <string>注释：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_product_version">
              <property name="text">
               <string>产品版本：</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEdit_company_name">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_product_name">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_file_desc">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_legal_copyr">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_legal_trademark">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_orig_filename">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_comment">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SizeConstraint::SetFixedSize</enum>
              </property>
              <item>
               <widget class="VersionEdit" name="lineEdit_product_version"/>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>文件版本：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="VersionEdit" name="lineEdit_fileversion"/>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>内部名称：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_inside_name">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VersionEdit</class>
   <extends>QLineEdit</extends>
   <header>advanced_wids</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
